cogs/triggered/triggered.py

```diff
@@ -0,0 +1,117 @@
+"""Triggered cog
+`triggered from spoopy.
+"""
+
+import os
+import urllib.request
+import discord
+from redbot.core import commands, data_manager
+from PIL import Image, ImageChops, ImageOps, ImageFilter, ImageEnhance
+
+AVATAR_URL = "https://cdn.discordapp.com/avatars/{0.id}/{0.avatar}.png?size=512"
+
+
+class Triggered(commands.Cog):
+    """We triggered, fam."""
+
+    # Class constructor
+    def __init__(self, bot):
+        self.bot = bot
+        self.saveFolder = data_manager.cog_data_path(cog_instance=self)
+
+    @commands.command(name="triggered")
+    async def triggered(self, ctx, user: discord.Member = None):
+        """Are you triggered? Say no more."""
+        if not user:
+            user = ctx.message.author
+        async with ctx.typing():
+            # bot is typing here...
+            savePath = await self._createTrigger(user, False, False)
+            if not savePath:
+                await self.bot.say("Something went wrong, try again.")
+                return
+            await ctx.send(file=discord.File(savePath))
+
+    @commands.command(name="reallytriggered")
+    async def hypertriggered(self, ctx, user: discord.Member = None):
+        """Are you triggered? Say no more."""
+        if not user:
+            user = ctx.message.author
+        async with ctx.typing():
+            # bot is typing here...
+            savePath = await self._createTrigger(user, False, True)
+            if not savePath:
+                await self.bot.say("Something went wrong, try again.")
+                return
+            await ctx.send(file=discord.File(savePath))
+
+    @commands.command(name="hypertriggered")
+    async def deepfry(self, ctx, user: discord.Member = None):
+        """Are you triggered? Say no more."""
+        if not user:
+            user = ctx.message.author
+        async with ctx.typing():
+            # bot is typing here...
+            savePath = await self._createTrigger(user, True, False)
+            if not savePath:
+                await self.bot.say("Something went wrong, try again.")
+                return
+            await ctx.send(file=discord.File(savePath))
+
+    async def _createTrigger(self, user, deepFry, hyper):
+        """Fetches the user's avatar, and creates a triggered GIF
+        Parameters:
+        -----------
+        user: discord.Member
+        deepFry: bool
+        hyper: bool
```

Comment:
Just a thought while I was eating breakfast, what are your thoughts on making `deepFry` and `hyper` an optional keyword argument like `mode`, and it takes in either "deepfry" or "hyper"? That way, when we use it above we'll know what it is without having to dive to the method definition to see which bool maps to hyper or deepfry. Would require changing the method to fix the bool stuff.
```suggestion
    async def _createTrigger(self, user: discord.Member, mode: str = None):
        """Fetches the user's avatar, and creates a triggered GIF
        
        Parameters:
        -----------
        user: discord.Member
        mode: Optional[str]
            One of the following: "hyper", "deepfry".
```